airbyte-integrations/connectors/source-s3/integration_tests/test_acceptance.py

```diff
@@ -192,40 +192,6 @@ def test_basic_read(instance: AcceptanceTestInstance) -> None:
         raise AssertionError("Expected records but got none.")  # noqa: TRY003
 
 
-@pytest.mark.parametrize(
```

Comment:
This test was failing and I can't figure out when it ever passed so I'm just going to remove it